### PR TITLE
feat: add Blitz.js support

### PR DESCRIPTION
**Pull request by Skn0tt**
Hi! The @blitz-js community would love to see support for Blitz on Netlify. This PR adds framework detection to setup the nextjs build plugin.

### CODE REVIEW

File: src/frameworks/blitz.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "excludedNpmDependencies": [],
+    "configFiles": []
+  },
+  "dev": {
+    "command": "blitz dev",
+    "port": 3000,
+    "pollingStrategies": [{ "name": "TCP" }]
+  },
+  "build": {
+    "command": "blitz buildd",
```
Comment (erezrokah) [inline src/frameworks/blitz.json:16]:
```suggestion
    "command": "blitz build",
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
This looks great @Skn0tt, per one comment which I assume is a typo.

For the plugins, I remember the Next.js plugin dynamically requiring `next` as a dependency https://github.com/netlify/netlify-plugin-nextjs/blob/7e02782bf3bd6d0a38267a1dde73737ddbc7c346/helpers/validateNextUsage.js#L34
will that work with Blitz.js?

**Comment by Skn0tt** [comment]
I haven't checked, but my gut says it won't work. An easy fix for the time being will be to spoof the node_modules/next/package.json file that the plug-in is checking for. That can be done by users themselves, until there's a better solution.

**Comment by erezrokah** [comment]
> I haven't checked, but my gut says it won't work. An easy fix for the time being will be to spoof the node_modules/next/package.json file that the plug-in is checking for. That can be done by users themselves, until there's a better solution.

Thanks for the quick reply. How about we decouple the plugin part from this PR? We can merge this without any recommended plugins, then work on finding a solution for making the Next.js plugin work with Blitz.js.

**Review by erezrokah — APPROVED** [review]

**Pull request merged**